src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -39,6 +39,9 @@ public boolean start() throws Exception {
         System.out.println(templates);
 
         Node node = computer.getNode();
+        if (node == null) {
+            throw new Exception("Computer does not correspond to a live node");
```

Comment:
I was thinking the plugin could tell the user that the node could not be live due to some unexpected conditions: the node might have been take off line, or might have been removed; but maybe I misunderstand what `getNode()` can provide in terms of information.

Another corner condition that I see is with the `Computer` object. If there are zero executors, then it won't exist (null?). Does it make sense for the plugin to cover the case of a non-existent `Computer` instance (even if the user is to blame)?